File: src/main/scala/com/github/kmizu/yapp/util/Closer.scala
package com.github.kmizu.yapp.util

import java.io.Closeable
import java.io.IOException

object Closer {
  def close(closable: Closeable): Unit = {
    if (closable != null) try {
      closable.close
    } catch {
      case e: IOException =>
        throw new RuntimeException(e)
    }
  }
}

File: src/main/java/com/github/kmizu/yapp/util/Closer.java
package com.github.kmizu.yapp.util;

import java.io.Closeable;
import java.io.IOException;

public class Closer {
  public static void close(Closeable closable) {
    if(closable != null)
      try {
        closable.close();
      } catch (IOException e) {
        throw new RuntimeException(e);
      }
  }
}
